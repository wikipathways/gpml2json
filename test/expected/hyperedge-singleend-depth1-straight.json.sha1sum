238c5729a6d880f422e1e8649fe340879182c041 ?-
