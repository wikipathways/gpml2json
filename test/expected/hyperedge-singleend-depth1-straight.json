{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["idaf905386","id28382a1c","faa53"],"drawAs":"rect","gpmlElementName":"Pathway","height":183.20987654320976,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":193.78600823045275,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130715","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/1f9a5e8b2e872fdbb604ca00c66770ebf6c7507227f5e5da6d653504ba782be2"},"entityMap":{"faa53":{"id":"faa53","textContent":"GeneProduct","wpType":"GeneProduct","x":66.17283950617399,"y":133.20987654320976,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"idaf905386":{"id":"idaf905386","borderWidth":1,"zIndex":12288,"points":[{"x":30.452674897119397,"y":66.6666666666666},{"x":163.78600823045275,"y":66.66666666666659}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","burrs":["e0874"]},"e0874":{"id":"e0874","attachmentDisplay":{"position":[0.5617283950617287,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"idaf905386","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id28382a1c":{"id":"id28382a1c","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"faa53","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":106.172839506174,"y":133.20987654320976,"orientation":[0,-1]},{"isAttachedTo":"e0874","attachmentDisplay":{"position":[0.5617283950617287,0],"offset":[],"relativeOffset":[0,0]},"x":105.34979423868322,"y":66.66666666666659,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimCatalysis","isAttachedTo":["faa53","idaf905386"]}}}
